website/source/docs/configuration/seal/gcpckms.html.md

```diff
@@ -81,14 +81,17 @@ The service account needs the following minimum permissions on the crypto key:
 ```text
 cloudkms.cryptoKeyVersions.useToEncrypt
 cloudkms.cryptoKeyVersions.useToDecrypt
+cloudkms.cryptKeys.get
 ```
 
-these permissions are available as part of the following role:
+These permissions can be described with the following role:
 
 ```text
 roles/cloudkms.cryptoKeyEncrypterDecrypter
+cloudkms.cryptKeys.get
```

Comment:
Should this actually be `cloudkms.cryptoKeys.get`?

Reply:
Sorry for typo, `cloudkms.cryptoKeys.get` is correct. I'll submit fix PR immediately.

Reply:
fixed typo: https://github.com/hashicorp/vault/pull/6515